docs/Board - ColibriRace.md

```diff
@@ -0,0 +1,127 @@
+	| 3   | nRESET         |                                              |
+	| 4   | SWCLK          |                                              | 
+	| 5   | Ground         |                                              |
+	| 6   | SWO/TDO        |                                              |
+
+### Internal Bus
+
+	| Pin | Function       | Notes                                        |
+	| --- | -------------- | -------------------------------------------- |
+	| 1   | PWM1	       | MOTOR 1                                       |
+	| 2   | PWM2           | MOTOR 2                		               |
+	| 3   | PWM3           | MOTOR 3                                      |
+	| 4   | PWM4           | MOTOR 4                                      | 
+	| 5   | PWM5 	       | MOTOR 5 (For Y6 or Hex X)                     | 
+	| 6   | PWM6 	       | MOTOR 6 (For Y6 or Hex X)                     | 
+	| 7   | BST SDA        | Use For TBS CorePro Control Device           |
+	| 8   | BST SCL        | Use For TBS CorePro Control Device           |
+	| 9   | PWM7           | Can be a normal GPIO (PA2) or PWM            |
+	| 10  | PWM8           | Can be a normal GPIO (PA2) or PWM            |
+	| 11  | 12.2V DCDC     | If 12v is detected, the Blue LED will trun on|
+	| 12  | 5.1V DCDC      | Voltage for MCU                              |
+
+### Servo
+
+	| Pin | Function       | Notes                                        |
+	| --- | -------------- | -------------------------------------------- |
+	| 1   | Ground         |                                              |
+	| 2   | VCC_OUT        | 5.1 Volt output to LCD Strip                 |
+	| 3   | PWM Servo      | PB14 - PWM10                                 |
+
+### IO_1 - LED Strip
+
+	| Pin | Function          | Notes                                        |
+	| --- | ----------------- | -------------------------------------------- |
+	| 1   | LED_STRIP         | Enable `feature LED_STRIP`                   |
+	| 2   | VCC_OUT           | 5.1 Volt output to LCD Strip                 |
+	| 3   | Ground            |                                              |
+
+### IO_2 - Sensor Interface
+
+	| Pin | Function          | Notes                                        |
+	| --- | ----------------- | -------------------------------------------- |
+	| 1   | VCC_OUT           | 4.7 Volt output to the device                |
+	| 2   | Ground            |                                              |
+	| 3   | UART3 TX          | GPS                                          |
+	| 4   | UART3 RX          | GPS                                          |
+	| 5   | SDA               | mag, pressure, or other i2c device           |
+	| 6   | SCL               | mag, pressure, or other i2c device           |
+
+### IO_3 - RC input
+
+	When RX_PPM/RX_SERIAL is used the IO_3 pinout is as follows.
+	if RX_SERIAL, Telemetry need to set to UART2 and choose your device.
```

Comment:
These two sentences could possibly be reworked so the grammar and intention is clearer.